src/services/ai.ts

```
@@ -0,0 +1,641 @@
+        prompt: prompt,
+        format: 'json',
+        stream: false,
+        options: {
+          temperature: 0.7,
+          top_p: 0.9,
+        },
+      });
+
+      const content = response.data.response;
+      return this.parseResponse(content, context.questionCount);
+    } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)